Directory.Build.props

```diff
@@ -0,0 +1,5 @@
+<Project>
+  <PropertyGroup>
+    <RazorPackageVersion>2.1.0-preview2-30159</RazorPackageVersion>
```

Comment:
I just grabbed a relatively new build. We can update this when we need new stuff. Due to the reliance on VS bits, we'll need to be a bit careful about it.

Ideally we can make improvements to a 'target' build of VS and then pull packages that include the same stuff after it makes its way into VS.

The alternative would be to install private builds of the Razor extension, which shouldn't be too bad either.

Reply:
Does this have any impact on people wanting to use this with VS 15.5? My guess is no impact since we're not yet involved in VS internally. For our first distribution in ~2 weeks we want to remain compatible with VS15.5.

Reply:
Your guess is correct, anyone can use the runtime or msbuild-based compiler with just .NET CLI.